lib/miq_automation_engine/models/miq_ae_domain.rb

```diff
@@ -103,6 +103,10 @@ def squeeze_priorities
     MiqAeDomain.reset_priority_by_ordered_ids(ids)
   end
 
+  def self.any_enabled?
```

Comment:
@hayesr can't we call any_editable_domains? here https://github.com/ManageIQ/manageiq/blob/master/app/models/tenant.rb#L238 instead of adding new method.